### PR TITLE
Wip

### PR DESCRIPTION
**Pull request by wozane**

### CODE REVIEW

**Comment by wozane** [comment]
Request for code review @yakato

File: stylesheet.css [hunk 1]
```diff
@@ -1,195 +1,1040 @@
...
+}
+
+body header #main-nav ul {
+  display: none;
+}
+
+body.menu-mobile-active {
+  overflow-y: hidden;
+  transition: 5s ease;
+  -webkit-transition: 5s ease;
```
Comment (yakato) [inline stylesheet.css]: It shouldn't be here anymore

File: stylesheet.css [hunk 2]
```diff
@@ -1,195 +1,1040 @@
...
+}
+
+body.menu-mobile-active #main-nav:before {
```
Comment (yakato) [inline stylesheet.css]: If in media queries you're specyfing this also with `header` selector, you should add it here too.

File: stylesheet.css [hunk 3]
```diff
@@ -1,195 +1,1040 @@
...
+  font-weight: 400;
+}
+
+#sumup-contact footer {
+  padding: 3rem 0;
+  font-size: 0.8rem;
+  line-height: 1.4;
+}
+
+/* Small devices (tablets, 768px and up) */
```
Comment (yakato) [inline stylesheet.css]: Add indents if you want to toggle code blocks with comment

File: stylesheet.css [hunk 4]
```diff
@@ -1,195 +1,1040 @@
...
+  }
+
+  body header #main-nav ul,
+  body.menu-mobile-active header #main-nav ul {
+    display: flex;
+    position: initial;
+  }
+
+  body header #main-nav ul li,
+  body.menu-mobile-active #main-nav ul li {
```
Comment (yakato) [inline stylesheet.css]: `header` selector is missing here

File: stylesheet.css [hunk 5]
```diff
@@ -1,195 +1,1040 @@
...
+    padding: 0;
+    margin: 0;
+    line-height: 5rem;
+    flex-grow: 1;
+    justify-content: space-between;
+    background-color: white;
+  }
+
+  body #main-nav ul li a,
+  body.menu-mobile-active #main-nav ul li a {
```
Comment (yakato) [inline stylesheet.css]: and here :) and probably below

File: stylesheet.css [hunk 6]
```diff
@@ -1,195 +1,1040 @@
...
+  }
+
+  #projects ul li:nth-child(2),
+  #projects ul li:nth-child(5) {
+    border-right: 1px solid #d6d6d6;
+    border-left: 1px solid #d6d6d6;
+  }
+
+
+
```
Comment (yakato) [inline stylesheet.css]: Empty lines here, remove please

**Review by yakato — CHANGES_REQUESTED** [review]
- On desktop there is something wrong with navbar after click on item in menu. Also there is noscroll effect then.
- On tablet there is bug when hover team meber, also when you click on it.
- Please add transition to mobile menu list

File: stylesheet.css [hunk 7]
```diff
@@ -81,8 +81,7 @@
 
   body.menu-mobile-active {
     overflow-y: hidden;
-    transition: 5s ease;
-    -webkit-transition: 5s ease;
+
   }
```
Comment (yakato) [inline stylesheet.css]: Empty line...

**Comment by wozane** [comment]
@yakato changes applied

**Review by yakato — APPROVED** [review]

**Pull request merged**